src/server.py

```diff
@@ -0,0 +1,93 @@
+import mysql.connector
+
+from json import dumps
+
+from sanic import Sanic
+from sanic.response import json
+
+
+app = Sanic()
+
+mydb = mysql.connector.connect(
+    host='db',
+    user='root',
+    passwd='password',
+    database='cat_db'
+)
+mycursor = mydb.cursor()
+
+
+@app.route('/get_db', methods=['GET'])
+async def get_db(request):
+    try:
+        mycursor.execute('SHOW DATABASES')
```

Comment:
Can you please use [aiomysql](https://github.com/aio-libs/aiomysql) instead of the `mysql-connector` since this is the tool we use in other projects?